src/webapi/element.rs

```diff
@@ -60,6 +60,20 @@ pub trait IElement: INode + IParentNode {
         ).unwrap()
     }
 
+    /// scrollTop
+    fn scroll_top( &self ) -> i32 {
+        js!(
+            return @{self.as_ref()}.scrollTop;
+        ).try_into().unwrap()
+    }
+
+    /// scrollTop
+    fn set_scroll_top( &self, value: u32 ) {
```

Comment:
Hmm... is there any particular reason why the getter returns `i32` and the setter accepts `u32`?

Reply:
@koute ah, good catch... I guess I should pick one 🤦‍♂️ 

what do you think makes the most sense here?

Reply:
In general we should stick with the types from the specs, so it'd be the best if you'd find the relevant IDL in the specs and use the type specified there. (:

Reply:
@koute the IDL states "unrestricted double"

Reply:
Unrestricted double should be `f64` no?
If you scroll down to `TypedArray objects` table, you can see what the Web IDL types represent.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray

Reply:
@rickycodes You can click on the type to get more information.

The [unrestricted double](https://heycam.github.io/webidl/#idl-unrestricted-double) type is 64-bit floating point, so the type should be `f64`

Reply:
ah, ok. thanks for clarification! I'll update in the PR

Reply:
Just ant to do some additional clarification... Here's some example code of mine:

Reply:
@rickycodes [The `double` type](https://heycam.github.io/webidl/#idl-double) should also be `f64`, therefore `page_y_offset` should return `f64`.

The only difference between `double` and `unrestricted double` is that in the WebIDL they guarantee that `double` will not be `NaN`, `-Infinity`, or `Infinity` (but the type is the same).

Reply:
Ok, cool. I just updated that as well. If things look good I can update with documentation/links and this should be good to go :)

Reply:
Yep. Still needs the docs and the links to the specs. (:

Reply:
Ok, this should be all good!